gooddata-pandas/gooddata_pandas/good_pandas.py

```diff
@@ -17,11 +17,23 @@ class GoodPandas:
     Facade to access factories that create pandas Series and DataFrames using analytics computed by GoodData.CN.
     """
 
-    def __init__(self, host: str, token: str, headers_host: Optional[str] = None) -> None:
-        self._sdk = GoodDataSdk.create(host, token, USER_AGENT, Host=headers_host)
+    def __init__(
+        self,
+        host: str,
+        token: str,
+        headers_host: Optional[str] = None,
+        **custom_headers_: Optional[str],
+    ) -> None:
+        if headers_host is not None:
+            custom_headers_["Host"] = headers_host
```

Comment:
IMHO this is not necessary. We can have Host=None. The GoodDataSdk filters out headers with None as a value, but if you want to keep it like this, we can.

Reply:
That's to retain backwards compatibility allowing use of headers_host variable. However I don't want to override the custom_headers_' "Host" entry when the headers_host is absent.

Reply:
Fair enough. Resolving conversation.